<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="ambientGroupBox">
       <property name="title">
        <string>ambient</string>
       </property>
       <widget class="QSlider" name="ambientRSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>51</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="ambientGSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>105</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>51</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>105</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>159</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>213</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
       <widget class="QSlider" name="ambientBSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>159</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="ambientASlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>213</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="diffuseGroupBox">
       <property name="title">
        <string>diffuse</string>
       </property>
       <widget class="QSlider" name="diffuseRSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>51</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="diffuseGSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>105</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>51</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>105</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>159</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>213</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
       <widget class="QSlider" name="diffuseBSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>159</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="diffuseASlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>213</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="specularGroupBox">
       <property name="title">
        <string>specular</string>
       </property>
       <widget class="QSlider" name="specularRSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>51</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="specularGSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>105</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>51</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>105</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>159</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>213</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
       <widget class="QSlider" name="specularBSlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>159</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="specularASlider">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>213</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="lightPosGroupBox">
       <property name="title">
        <string>lightPos</string>
       </property>
       <widget class="QDoubleSpinBox" name="lightPosXSpinBox">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>22</y>
          <width>58</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lightPosYSpinBox">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>48</y>
          <width>58</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>48</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>74</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lightPosZSpinBox">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>74</y>
          <width>58</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
    <zorder>lightPosGroupBox</zorder>
    <zorder>ambientGroupBox</zorder>
    <zorder>diffuseGroupBox</zorder>
    <zorder>specularGroupBox</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
